server/src/utils/validations/createBlog.js

```diff
@@ -0,0 +1,21 @@
+import isEmpty from './isEmpty';
+
+function validateBlogInput(blog) {
+  const errors = {};
+
+
+  if (isEmpty(blog.title)) {
+    errors.title = 'Title is required';
+  }
+
+  if (isEmpty(blog.content)) {
+    errors.content = 'Content is required';
+  }
+
+  return {
+    errors,
+    isValid: isEmpty(errors),
+  };
+}
+
+export default validateBlogInput;
```

Comment:
Refer to the sign-in validation. You can create this in just 4 lines.

Reply:
@piyushagwl007 1) That is unnecessary abstraction, I feel.
Also I have referenced your signup validation :P

Reply:
I forgot to change that in the signup validation :face_with_head_bandage:     But I guess that is not unnecessary because in here you are repeating the code to add error into the error object.